lib/content-services/src/lib/common/services/rendition.service.ts

```diff
@@ -201,7 +201,7 @@ export class RenditionService {
                                 clearInterval(intervalId);
                                 return resolve(this.handleNodeRendition(nodeId, rendition.entry.content.mimeType, versionId));
                             }
-                        }, () => reject());
+                        }, () => reject(new Error('error')));
```

Comment:
Should we provide something more meaningful as a error message here?

Reply:
added some messages